### PR TITLE
Migrate to ESM

### PR DESCRIPTION
**Pull request by irdkwmnsb**
Migrating almost everything to ESM.

Some things might have broken. Need to test thoroughly.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -103,7 +104,7 @@
     "key-spacing": [2, {"beforeColon": false, "afterColon": true, "mode": "strict"}], // optionally set `[2, {"beforeColon": false, "afterColon": true, "mode": "strict", "align": "colon"}]`
     "lines-around-comment": 2, // optionally set `[2, {"beforeBlockComment": true, "beforeLineComment": true, "allowBlockStart": true}]`
     "linebreak-style": 0, // optionally set `[1, "unix"]`
-    "max-len": [2, 150, 2, {"ignoreComments": true, "ignoreUrls": true}], // NOTE: Our limit is 80 however ESLint does not have an ignoreStrings so lets have more buffer
+    // stupid rule. "max-len": [2, 150, 2, {"ignoreComments": true, "ignoreUrls": true}], // NOTE: Our limit is 80 however ESLint does not have an ignoreStrings so lets have more buffer
```
Comment (DaniilSmirnov) [inline .eslintrc]: Может просто удалим?

File: lib/units/api/index.js [hunk 2]
```diff
@@ -1,156 +1,150 @@
...
+        subdev.subscribe(channel)
+    })
+    sub.on('message', function(channel, data) {
+        channelRouter.emit(channel.toString(), channel, data)
+    })
+    subdev.on('message', function(channel, data) {
+        channelRouter.emit(channel.toString(), channel, data)
+    })
+    // Swagger Express Config
+    app.use(bodyParser.json())
```
Comment (DaniilSmirnov) [inline lib/units/api/index.js:94]: А оно точно еще нужно?

Comment (irdkwmnsb) [reply lib/units/api/index.js:94]: Yes, express-openapi does not parse the body on itself.

**Comment by irdkwmnsb** [comment]
After many days and nights this merge request brings ESM to VKCOM's devicehub.  
We got rid of swagger-express-mw-node12 in favour of express-openapi which is a lot more modern and what's more important: supports ESM.  
ESM is available everywhere except the frontend package.  
There are also some minor chores and small bug fixes and chores but I don't have the resources to explain them right now and we don't yet have a change log.

**Review by DaniilSmirnov — APPROVED** [review]

**Pull request merged**